spec/features/support/searching_schools_spec.rb

```diff
@@ -71,6 +71,30 @@
     and_the_csv_contains_data_for_the_correct_schools
   end
 
+  context 'export of users scoped to schools' do
+    let(:school_user1) { create(:school_user, email_address: 'user@can-order-school.sch.uk', school: schools_who_can_order.first) }
+    let(:school_user2) { create(:school_user, email_address: 'user@another-can-order-school.sch.uk', school: schools_who_can_order.last) }
+    let(:trust_user) { create(:trust_user, email_address: 'user@trust.gov.uk', responsible_body: responsible_body) }
+    let(:out_of_scope_school_user) { create(:school_user) }
```

Comment:
Do we need a let! here or hit in a before block, otherwise this isn't actually there during the export and only created at the expectation?

Reply:
Good spot! `out_of_scope_school_user` was a late addition and wasn't included in the method `and_there_are_school_and_responsible_body_users`.

Reply:
Rubocop did not like `let!` so I stuck it into a before block